.docker/dev/php.ini

```diff
@@ -0,0 +1,12 @@
+[PHP]
+memory_limit=512M
+
+[date]
+date.timezone=${PHP_DATE_TIMEZONE}
+
+[xdebug]
```

Comment:
Is xdebug installed in this configuration? I don't see xdebug installation in https://github.com/Sylius/Sylius-Docker-Standard/blob/master/traditional/Dockerfile

Reply:
@coldic3 I added an explanation in the PR description. I considered removing xdebug configs, but It does not bother me XD What do you think should I remove it until next PR with xdebug?
> I decided to not install xdebug yet, but it will be provided in the nearest future

Reply:
It doesn't bother me as well but it may be a little bit misleading if someone like me didn't read PR description XD
Anyway thanks for clarifying that :P

Reply:
We tend to keep source code free of unused configs. Would be nice to remove it until next PR